pkg/fanal/image/daemon/image_test.go

```diff
@@ -296,7 +296,7 @@ func Test_image_emptyLayer(t *testing.T) {
 			history: dimage.HistoryResponseItem{
 				CreatedBy: "/bin/sh -c #(nop) WORKDIR /",
 			},
-			want: true,
+			want: false,
```

Comment:
`WORKDIR /` will be empty in Docker now. Is it correct? I thought it should be true always in case of Docker/Podman.